server/controllers/documents.js

```diff
@@ -0,0 +1,135 @@
+import { Documents } from '../models/';
+
```

Comment:
You should leverage es6 as much as possible. Basically, avoid the module pattern here and make use of es6 classes for all your controllers